pkg/webui/console/views/application/index.js

```diff
@@ -105,21 +106,21 @@ export default class Application extends React.Component {
   }
 
   render () {
-    const { fetching, error, match } = this.props
+    const { fetching, error, match, application } = this.props
 
-    if (fetching) {
+    // show any application fetching error, e.g. not found, not rights, etc
+    if (error) {
```

Comment:
Any decision here?

Reply:
removing it will crash the app on, say 404.
i cannot add **proper** error handling, because this https://github.com/TheThingsNetwork/lorawan-stack/issues/41 issue is still open and dont remeber us agreeing on **proper** error handling

Reply:
Ok, leave it as it is but as long as we don't have proper error handling I would actually prefer it to crash rather than the error getting swallowed with an "ERROR" text in UI.